src/shadowbox/model/metrics.ts

```diff
@@ -28,3 +28,15 @@ export interface DataUsageByUser {
 export interface DataUsageTimeframe {
   hours: number;
 }
+
+
+export interface TunneltimeMetrics {
```

Comment:
The response should contain a list of values. Each value is a duration, with dimensions. The client code can then iterate over the entries, aggregating accordingly.

Example:
```yml
[
  {key: key1, country: IR, asn: 1234, tunnel_time: {hours: 10}},
  {key: key1, country: IR, asn: 5678, tunnel_time: {hours: 2}}
]
```